### ISSUE TITLE
Color contrast test not working?

### ISSUE THREAD

**Issue by tounsoo** [NickColley/jest-axe#258]
Sample of my test:
```
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { axe, toHaveNoViolations } from 'jest-axe';

expect.extend(toHaveNoViolations);
afterEach(cleanup);

test('button(button) should pass accessiblity test', async () => {
    const { container } = render(<div style={{ backgroundColor: "#fff"}}><button style={{backgroundColor: '#fff', color: '#fff'}}>test</button></div>);
    expect(await axe(container)).toHaveNoViolations();
});
```

I get a pass with above code and am not getting violation for color contrast. Does `jest-axe` not have color contrast test?

**Comment by NickColley** [comment]
Axe does not support colour contrast tests in. Node.js environment (jsdom)

**Comment by tounsoo** [comment]
@NickColley If I want to also test for color contrast, what can I do? Should I use axe from axe-core for it?

**Comment by NickColley** [comment]
You'll want to run axe using a different approach that runs a full browser environment. Rather than jsdom. So Google puppeteer axe, webdriver axe etc
